### PR TITLE
Feature/#14 home post list

### LINKED ISSUE #14: view_home_post_list_feature

### PR DESCRIPTION
**Pull request by markusviola**
closes #14

### CODE REVIEW

File: app/Http/Controllers/HomeController.php [hunk 1]
```diff
@@ -24,6 +25,7 @@ public function __construct()
      */
     public function index()
     {
-        return view('home');
+        $posts = BlogPost::all();
```
Comment (andy1dx) [inline app/Http/Controllers/HomeController.php]: Add the ordering by created date desc

File: resources/views/home.blade.php [hunk 2]
```diff
@@ -3,20 +3,34 @@
...
+                <p>
+                <div class="text-muted">Posted by {{ $post->user->username }}</div>
+                <hr>
+                
+            @endforeach
         </div>
     </div>
 </div>
 @endsection
+
```
Comment (andy1dx) [inline resources/views/home.blade.php]: delete not use code

**Review by andy1dx — CHANGES_REQUESTED** [review]
Please fix

**Review by hugh-n — COMMENTED** [review]
Andy has already commented so I will review after the fixes are made.

File: app/Http/Controllers/HomeController.php (hunk 1 above)
Comment (andy1dx) [reply app/Http/Controllers/HomeController.php]: @markusviola  Make The order

File: resources/views/home.blade.php (hunk 2 above)
Comment (markusviola) [reply resources/views/home.blade.php]: done

File: app/Http/Controllers/HomeController.php (hunk 1 above)
Comment (markusviola) [reply app/Http/Controllers/HomeController.php]: done

**Review by andy1dx — APPROVED** [review]
LGTM

**Pull request merged**